pages/common/telnet.md

```diff
@@ -1,19 +1,19 @@
 # telnet
 
-> Telnet is used to connect to a specified port of a host.
+> Connect to a specified port of a host using the telnet protocol.
 
 - Telnet to a certain port:
```

Comment:
Not sure how I missed this comment -- probably the notification linked directly to the second comment you made. Yes, agree, and will make this change as soon as we reach consensus in the second thread below.